models/issue.go

```diff
@@ -745,6 +766,40 @@ func (issue *Issue) ChangeTitle(doer *user_model.User, oldTitle string) (err err
 	return committer.Commit()
 }
 
+// ChangeConfidential changes the confidential of this issue, as the given user.
+func (issue *Issue) ChangeConfidential(doer *user_model.User) (err error) {
```

Comment:
`ChangeConfidential` reads a little strange. Something like `TogglePrivate` in my mind.

(Since we already used the word `private`, maybe we shouldn't introduce a new concept `confidential`.

Reply:
`Private` it is.